_config.yml

```diff
@@ -2,13 +2,62 @@
 domain: services.github.com
 author: GitHub, Inc.
 email: services@github.com
-title: On Demand Training
+
+
+# Site Settings
+locale                   : "en-US"
+# Keep original, do not replace
+title                    : "On Demand Training"
+title_separator          : "-"
+name                     : "GitHub, Inc."
+description              : "You're in control. We're with you along the way."
+url                      : "https://github.github.com" # the base hostname & protocol for your site e.g. "https://mmistakes.github.io"
+#url                      : ""
+baseurl                  : "/on-demand" # the subpath of your site, e.g. "/blog"
+gh_repo                  :
```

Comment:
Possibly, let's do it in a separate PR though we can carefully explore what it does with that info, exactly.